Packs/ThalesCipherTrustManager/Integrations/ThalesCipherTrustManager/ThalesCipherTrustManager.yml

```diff
@@ -0,0 +1,2389 @@
+      description: Start date of the CA's certificate validity.
+      type: Date
+    - contextPath: CipherTrust.CAInstall.notAfter
+      description: End date of the CA's certificate validity.
+      type: Date
+    - contextPath: CipherTrust.CAInstall.sha1Fingerprint
+      description: SHA1 fingerprint of the CA's certificate.
+      type: String
+    - contextPath: CipherTrust.CAInstall.sha256Fingerprint
+      description: SHA256 fingerprint of the CA's certificate.
+      type: String
+    - contextPath: CipherTrust.CAInstall.sha512Fingerprint
+      description: SHA512 fingerprint of the CA's certificate.
+      type: String
+    - contextPath: CipherTrust.CAInstall.purpose.client_authentication
+      description: Indicates if client authentication is enabled for the CA.
+      type: String
+    - contextPath: CipherTrust.CAInstall.purpose.user_authentication
+      description: Indicates if user authentication is enabled for the CA.
+      type: String
+  - deprecated: false
+    description: Returns a list of local CA certificates. The results can be filtered, using the command arguments. If local_ca_id is provided, a single local CA certificate is returned and the rest of the filters are ignored. Chained parameter is used to return the full CA chain with the certificate and can be used only if local_ca_id is provided.
+    name: ciphertrust-local-ca-list
+    arguments:
+    - name: subject
+      isArray: false
+      description: Filter by subject
+      required: false
+      secret: false
+      default: false
+    - name: local_ca_id
+      isArray: false
+      description: An identifier of the resource. This can be either the ID (a UUIDv4),the Name, the URI, or the slug (which is the last component of the URI).
+      required: false
+      secret: false
+      default: false
+    - name: chained
+      isArray: false
+      description: When set to ‘true’ the full CA chain is returned with the certificate.Must be used with the local CA ID.
+      required: false
+      secret: false
+      default: false
+      predefined:
+      - 'true'
+      - 'false'
+      auto: PREDEFINED
+    - name: issuer
+      isArray: false
+      description: Filter by issuer
+      required: false
+      secret: false
+      default: false
+    - name: state
+      isArray: false
+      description: Filter by state
+      required: false
+      secret: false
+      default: false
+      predefined:
+      - pending
+      - active
+      auto: PREDEFINED
+    - name: cert
+      isArray: false
+      description: Filter by cert
+      required: false
+      secret: false
+      default: false
+    - name: page
+      isArray: false
+      description: Page to return.
+      required: false
+      secret: false
+      default: false
+    - name: page_size
+      isArray: false
+      description: Number of entries per page. Defaults to 2000 (in case only page was provided). Maximum entries per page is 2000
```

Comment:
```suggestion
      description: Number of entries per page. Defaults to 2000 (in case only page was provided). Maximum entries per page is 2000.
```